incubator/kubeless/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-controller
+  namespace: kubeless
```

Comment:
Why is the namespace hardcoded?

Reply:
Because is the namespace where kubeless expect to run. 
It can run on any other namespace but the user will have to specify it in every command.

I deleted the default namespace and added a note in the README.